_inc/client/plans/index.jsx

```diff
@@ -12,35 +13,204 @@ import PlanGrid from './plan-grid';
+								{ __( '{{strong}}Looking for more?{{/strong}}', {
+									components: {
+										strong: <strong />,
+									},
+								} ) }
+							</p>
+							<p>
+								{ __(
+									'With Real-time backups, we save as you edit and you’ll get unlimited backup archives.'
+								) }
+							</p>
+						</>
+					),
+					purchase,
+					isCurrent: true,
+				};
+			case 'jetpack_backup_realtime':
+				return {
+					title: __( 'Jetpack Backup {{em}}Real-Time{{/em}}', {
+						components: {
+							em: <em />,
+						},
+					} ),
+					subtitle: __( 'Purchased %(purchaseDate)s', {
+						args: {
+							purchaseDate: moment( purchase.subscribedDate ).format( 'YYYY-MM-DD' ),
+						},
+					} ),
+					description: __(
+						'Always-on backups ensure you never lose your site. Your changes are saved as you edit and you have unlimited backup archives.'
+					),
+					purchase,
+					isCurrent: true,
+				};
+			case 'is-personal-plan':
+				return {
+					title: __( 'Jetpack Backup {{em}}Daily{{/em}}', {
+						components: {
+							em: <em />,
+						},
+					} ),
+					subtitle: __( 'Included in your {{planLink}}Personal Plan{{/planLink}}', {
+						components: {
+							planLink: <a href={ `/plans/my-plan/${ siteRawlUrl }` } />,
+						},
+					} ),
+					description: __( 'Always-on backups ensure you never lose your site.' ),
+					purchase,
+					isCurrent: true,
+				};
+			case 'is-premium-plan':
+				return {
+					title: __( 'Jetpack Backup {{em}}Daily{{/em}}', {
+						components: {
+							em: <em />,
+						},
+					} ),
+					subtitle: __( 'Included in your {{planLink}}Premium Plan{{/planLink}}', {
+						components: {
+							planLink: <a href={ `/plans/my-plan/${ siteRawlUrl }` } />,
+						},
+					} ),
+					description: __( 'Always-on backups ensure you never lose your site.' ),
+					purchase,
+					isCurrent: true,
+				};
+			case 'is-business-plan':
+				return {
+					title: __( 'Jetpack Backup {{em}}Real-Time{{/em}}', {
+						components: {
+							em: <em />,
+						},
+					} ),
+					subtitle: __( 'Included in your {{planLink}}Professional Plan{{/planLink}}', {
+						components: {
+							planLink: <a href={ `/plans/my-plan/${ siteRawlUrl }` } />,
+						},
+					} ),
+					description: __(
+						'Always-on backups ensure you never lose your site. Your changes are saved as you edit and you have unlimited backup archives.'
+					),
+					purchase,
+					isCurrent: true,
+				};
+		}
+	}
+
+	getTitleSection() {
+		return (
+			<Fragment>
+				<h1 className="plans-section__header">{ __( 'Solutions' ) }</h1>
+				<h2 className="plans-section__subheader">
+					{ __( "Just looking for backups? We've got you covered." ) }
+				</h2>
+			</Fragment>
+		);
+	}
+
+	findPrioritizedPurchase() {
+		const { activeSitePurchases } = this.props;
+
+		// Note: the order here is important, as it resolves cases where a site
+		// has both a plan and a product at the same time.
+		const planClasses = [
+			'is-business-plan',
```

Comment:
Do we need to take VIP and eCommerce plans into account here?

Reply:
They are taken in to account here by using the plan class, see the implementation of `getPlanClass` [here](https://github.com/Automattic/jetpack/blob/master/_inc/client/lib/plans/constants.js#L122) to see how.